Utilities/Checks.py

```diff
@@ -0,0 +1,24 @@
+import discord

```

Comment:
I'm not sure what this file does exactly can you add a comment at the top of the code to explain this code's
purpose